iron-list.html

```diff
@@ -882,7 +885,8 @@
       var physicalCount = this._physicalCount;
       var top = this._physicalTop + this._scrollOffset;
       var bottom = this._physicalBottom + this._scrollOffset;
-      var scrollTop = this._scrollTop;
+      // This may be called outside of a scrollHandler, so use last cached position
+      var scrollTop = this._scrollPosition;
```

Comment:
Found the comment confusing. Isn't `_scrollPosition` the more "live" of the two properties (`_scrollPosition` and `_scrollTop`)?

Reply:
Actually `_scrollTop` is "live." It comes from `iron-scroll-target-behavior`.

Reply:
Added comment to declaration of `_scrollPosition` that it is a cached version of `_scrollTop`